### ISSUE TITLE
Take `Nette\Bootstrap\Configurator::$staticParameters` into account

### ISSUE THREAD

**Issue by dakur** [mesour-intellij-plugins/neon-nette-issues#16]
**Is your feature request related to a problem? Please describe.**
When you Ctrl+click on a parameter defined in .neon file, it correctly jumps to the place where it's defined. However when you Ctrl+click on a parameter which is defined while bootstraping application (`Nette\Bootstrap\Configurator::$staticParameters` property), it doesn't jump anywhere.

**Describe the solution you'd like**
Jump to the place, where the param is defined.

I would use it when I was searching for what the `tempDir` is. But I'm not pretty sure if it's possible actually, just a suggestion. 🙂

**Comment by mabar** [comment]
Seems bit too complex for little added value. Maybe nette/bootstrap could add a stub file?

```
# Stub file for IDE neon support
parameters:

	# Configurator parameters
	appDir: /app
	logDir: /log
	tempDir: /tmp
	vendorDir: /vendor
	wwwDir: /www
	debugMode: false
	consoleMode: false
```

**Comment by dakur** [comment]
Yeah, that makes sense, Nette should keep params up-to-date. I can make PR to nette/bootstrap.

**Comment by mesour** [comment]
You can use XML configuration in 1.3.0 - https://mesour-intellij-plugins.github.io/#xml-config
